src/components/Card/Card.css

```diff
@@ -79,6 +79,10 @@
   text-transform: lowercase;
 }
 
+.card .tag:hover {
+  cursor: pointer;
```

Comment:
This can actually be placed in `.card .tag`. No need to use the `:hover` selector.

https://jsfiddle.net/DRFR0ST/t8fpd6nw/

Reply:
Sure :) But I'm not sure what the benefit to set it on the regular state..

Reply:
It's less efficient/ergonomic to create pseudo-classes for one property instead of adding it to the regular state and in this case it saves a few lines of code. Anyway you're right, it's nothing one should worry about as it doesn't really make a difference in performance.

Reply:
Maybe in old browsers. I remember something about old IEs that didn't listen to element click event without `cursor: pointer`. Some weird bug.

Reply:
Safari in mobile used to behave the same @moshfeu

Reply:
Tags should be `<button>` not divs. Please.

Reply:
@equinusocio good point!
Just need to look as the the way it is now.